STREAMS-1734 document Streams metrics to monitor (#100)
Axway/streams-open-docs: content/en/docs/metrics.md

```sh
rate(http_server_requests_seconds_sum[2m])/rate(http_server_requests_seconds_count[2m])
```

## Monitoring

Regardless of basic system metrics (as CPU usage or memory usage), Streams expose custom metrics that need to be monitored to prevent issues with your Streams installation:

> * number of active subscription:

Reviewer: ```suggestion
* Number of active subscriptions:
```

    * it must still under 1500 for each subscriber pods
    * it's given by `streams_active_subscriptions`

> * rate of events in input in Streams topics:

Reviewer: ```suggestion
* Rate of input events for Streams hub:
```

    * it must still under 500 events by seconds
    * it's given by `rate(streams_input_events_total{streams_service="hub", data_type="snapshot"}[2m])`
* JVM memory used:

>     * it must still lower than the total pod memory for each pods, or they will be OOM kill after a short time:

Reviewer: ```suggestion
    * it must still lower than the total pod memory for each pods, otherwise they will be OOM kill after a short period of time
```
